### DISCUSSION TITLE
Combine API Rules Users and Admin

### DISCUSSION THREAD

**Discussion by ibamibrhm** [pocketbase/pocketbase#3940] [Q&A] [answered]
Hello, I'm wondering if it's possible to merge API rules to permit `@request.auth.role = "XXX"` alongside Admin access. I'm currently utilizing the API directly through axios in a Node.js backend. Is there a method to achieve this?

**Comment by ganigeorgiev** [comment, marked as answer]
I don't understand what do you mean by _'merge API rules to permit @request.auth.role = "XXX" alongside Admin access'_. API rules don't apply for admins.

**PocketBase Admins can access and modify everything (including deleting or creating other admins).**

> currently utilizing the API directly through axios in a Node.js backend

Unless you want to support older Node.js versions, there isn't much benefit of using `axios` instead of directly `fetch` or the [JS SDK](https://github.com/pocketbase/js-sdk) (_under the hood it uses `fetch`_).
